mbed_lstools/main.py

```diff
@@ -170,8 +170,8 @@ def get_mbedls_version():
     mbeds.debug(__name__, "mbed-ls ver. " + get_mbedls_version())
     mbeds.debug(__name__, "host: " +  str((mbed_lstools_os_info())))
 
-    if not opts.skip_retarget:
-        mbeds.retarget()
+    if opts.skip_retarget:
+        mbeds.retarget(False)
```

Comment:
Based on my proposed changes above, you'd instead pass a parameter to the `create` function above that enables/disables retargeting. That way the appropriate parameter can be passed to the constructor of the `MbedLsTools*` class.